api/requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.11
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
Noticed this and wanted to highlight it – you might want to check your local Python env.

Reply:
Ah yep, good catch. Thanks!

Reply:
I've updated my environment and re run `pip-compile` just in case there were any differences.